packages/examples/package.json

```diff
@@ -20,7 +20,7 @@
   "sideEffects": false,
   "files": [],
   "scripts": {
-    "build": "yarn workspaces foreach --worktree --parallel --verbose --no-private run build",
+    "build": "yarn workspaces filter --include 'packages/examples/packages/**' --parallel --no-private run build",
```

Comment:
What's the difference between doing `yarn workspaces filter --include` and `yarn workspaces foreach --include`? Why we don't need `--verbose` any more?

Reply:
`--verbose` doesn't exist on `filter`

Reply:
`workspaces filter`'s `--include` supports glob patterns, whereas `workspaces foreach` only supports exact package names. `--verbose` is also enabled by default on `workspaces filter`.